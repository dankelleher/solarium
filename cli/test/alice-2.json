[119,224,94,206,1,200,147,17,235,153,132,142,157,120,231,161,99,172,148,201,6,20,174,39,78,254,30,196,182,152,21,250,118,241,201,156,200,185,43,210,159,160,179,86,4,250,134,187,161,8,101,181,195,130,50,181,216,50,217,103,53,101,235,165]