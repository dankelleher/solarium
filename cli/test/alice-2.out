Generating a new keypair

Wrote new keypair to alice-2.json
=============================================================================
pubkey: 91JwtvZQtK4jGpRMTwssK4Sfc8L9PQY5NKPgM4qGCLep
=============================================================================
Save this seed phrase and your BIP39 passphrase to recover your new keypair:
essence sunny obvious guilt exit punch kind flash purchase rule abstract omit
=============================================================================
